skore/src/skore/cross_validate.py

```diff
@@ -246,9 +246,20 @@ def cross_validate(*args, project: Optional[Project] = None, **kwargs) -> dict:
     cv_results = sklearn.model_selection.cross_validate(
         *args, **kwargs, scoring=new_scorers
     )
+
     cross_validation_item = CrossValidationItem.factory(cv_results, estimator, X, y)
 
     if project is not None:
+        try:
+            cv_results_history = project.get_item_versions("cross_validation")
+        except KeyError:
+            cv_results_history = []
+
+        agg_cross_validation_item = AggCrossValidationItem.factory(
+            cv_results_history + [cross_validation_item]
+        )
+
+        project.put_item("cross_validation_aggregated", agg_cross_validation_item)
```

Comment:
Could this item key be parametrized as a kwarg ? I suppose that a user may cross validate multiple models or just customize / put in a "folder" there results.

Reply:
This is future work because we want to avoid changing the signature of `cross_validate` too much yet (we're discussing this kind of things a lot these days)

Reply:
Alright then !